### PR TITLE
Add "backup" tool

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add a script to backup your files to ClickHouse. This is strange, but works.

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit 3b8adbb8def6ea1b3f3b90be46634255b989bca7 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/65699/3b8adbb8def6ea1b3f3b90be46634255b989bca7/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Cloud fork sync (only for ClickHouse Inc. employees)</td><td>If it fails, ask a maintainer for help</td><td>⏳ pending</td></tr>
<tr><td>PR Check</td><td>Checks correctness of the PR's body</td><td>❌ failure</td></tr>
<tbody>
</table>

File: utils/backup/backup [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    exit 1
+}
+
+while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+        --host)
+            host="$2"
+            shift 2
+            ;;
+        --user)
```
Comment (thevar1able) [inline utils/backup/backup:21]: Should table also be configurable?

**Review by thevar1able — APPROVED** [review]

**Comment by vitlibar** [comment]
Maybe you can name your tool not `backup` but something else?
Because the `backup` already means something in ClickHouse ;)

File: utils/backup/backup (hunk 1 above)
Comment (alexey-milovidov) [reply utils/backup/backup:21]: Probably yes, but so far, there is only a single use-case for this tool (I want to backup my files), so no need for prematurly adding features.

**Comment by alexey-milovidov** [comment]
@vitlibar, Yes, but I don't think it is a problem - this is just a script in `utils`, the chance of confusion is minimal.

File: utils/backup/backup (hunk 1 above)
Comment (thevar1able) [reply utils/backup/backup:21]: Understandable.

**Pull request merged**